.github/workflows/test-all-warehouses.yml

```diff
@@ -44,3 +44,35 @@ jobs:
       dbt-version: ${{ inputs.dbt-version }}
       generate-data: ${{ inputs.generate-data || false }}
     secrets: inherit
+
+  notify_failures:
```

Comment:
Not sure I understand [this comment](https://github.com/elementary-data/elementary/pull/1082#issuecomment-1680713605). Why not reference the job in the different workflow? like we do here with `test-warehouse.yml`

Reply:
I need to pass the jobs status to the uses and as far as I know its not possible
I can upload an artifact but at this point it might be an overhead for just those two things